k8s/upgrade/src/bin/upgrade-job/main.rs

```diff
@@ -41,7 +41,9 @@ async fn main() -> Result<()> {
 fn init_logging() {
     let tags = default_tracing_tags(raw_version_str(), env!("CARGO_PKG_VERSION"));
 
-    init_tracing("upgrade-job", tags, None);
+    TracingTelemetry::builder()
+        .with_tracing_tags(tags)
```

Comment:
missing format things from cli args? Similar to your changes on control-plane repo